src/scores.jl

```diff
@@ -287,6 +287,75 @@ function get_scores_at_step(all_scores::Array{Dict{String, Any}, 1},
 end
 
 
+function get_score_values(all_scores::Array{Dict{String, Any}, 1},
+                          score::Union{DistrictAggregate, DistrictScore};
+                          nested_key::Union{String,Nothing}=nothing)::Array
+    """ Returns the value of specified DistrictScore/DistrictAggregate score
+        at every step of the chain.
+
+        Arguments:
+            all_scores  : List of scores of partitions at each step of
+                         the Markov Chain
+            score       : DistrictScore of interest
+            nested_key  : If the score is nested within a CompositeScore, this
+                          argument provides the CompositeScore's name
+    """
+    # check if score is nested inside a CompositeScore
+    nested = nested_key == nothing
```

Comment:
im confused ... so if no `nested_key` is passed, then `nested = True`?

Reply:
shouldn't it be the other way round?  where if you pass in the Composite Score name, then `nested` should be `True`?

Reply:
Yep, fixed in 5fec88e! Thank you!